x-pack/plugin/profiling/src/main/java/org/elasticsearch/xpack/profiling/ProfilingIndexTemplateRegistry.java

```diff
@@ -47,13 +47,14 @@ public class ProfilingIndexTemplateRegistry extends IndexTemplateRegistry {
     // version 4: Added 'service.name' keyword mapping to profiling-events
     // version 5: Add optional component template '<idx-name>@custom' to all index templates that reference component templates
     // version 6: Added 'host.arch' keyword mapping to profiling-hosts
-    public static final int INDEX_TEMPLATE_VERSION = 6;
+    // version 7: Added 'host.type', 'cloud.provider', 'cloud.region' keyword mappings to profiling-hosts
+    public static final int INDEX_TEMPLATE_VERSION = 7;
 
     // history for individual indices / index templates. Only bump these for breaking changes that require to create a new index
     public static final int PROFILING_EVENTS_VERSION = 2;
     public static final int PROFILING_EXECUTABLES_VERSION = 1;
     public static final int PROFILING_METRICS_VERSION = 1;
-    public static final int PROFILING_HOSTS_VERSION = 1;
+    public static final int PROFILING_HOSTS_VERSION = 2;
```

Comment:
I decided for a roll-over since it's easier to argue about changes to be done in the collector.